### ISSUE TITLE
[2015.8.0rc1] Installation fails on stock Centos/RHEL 6 due to pycrypto 2.6 dependency

### ISSUE THREAD

**Issue by tbaker57** [saltstack/salt#25502]
As discussed in #25456 installing 2015.8.0rc1 via salt-bootstrap on a stock Centos 6 fails with
```
Starting salt-minion daemon:                               [  OK  ]
Starting salt-master daemon: Traceback (most recent call last):
  File "/usr/bin/salt-master", line 22, in <module>
    salt_master()
  File "/usr/lib/python2.6/site-packages/salt/scripts.py", line 47, in salt_master
    master.start()
  File "/usr/lib/python2.6/site-packages/salt/cli/daemons.py", line 150, in start
    self.prepare()
  File "/usr/lib/python2.6/site-packages/salt/cli/daemons.py", line 131, in prepare
    self.master = salt.master.Master(self.config)
  File "/usr/lib/python2.6/site-packages/salt/master.py", line 326, in __init__
    SMaster.__init__(self, opts)
  File "/usr/lib/python2.6/site-packages/salt/master.py", line 100, in __init__
    self.master_key = salt.crypt.MasterKeys(self.opts)
  File "/usr/lib/python2.6/site-packages/salt/crypt.py", line 205, in __init__
    self.key = self.__get_keys()
  File "/usr/lib/python2.6/site-packages/salt/crypt.py", line 256, in __get_keys
    key = RSA.importKey(f.read())
NameError: global name 'RSA' is not defined
```
As suggested by @s0undt3ch this is due to the fact that 2015.8 now requires pycrypto >=2.6. However stock Centos/RHEL 6 only comes with python-crypto-2.0.1. (Centos 7 has pycrypto 2.6 in the base repo so this shouldn't be an issue for Centos 7. Centos 5  - hmmm, good luck there!)

python-crypto2.6  is available in epel, but I couldn't get it to work out of the box due to the egg structure (sorry, I'm not a python person so I don't understand the module loading mechanism in python and/or how python rpms are meant to be packaged). To get it to work (on x64), I needed to do the following
```
yum erase -y  python-crypto
yum install -y   python-crypto2.6   # make sure epel is installed
ln -s /usr/lib64/python2.6/site-packages/pycrypto-2.6.1-py2.6-linux-x86_64.egg/Crypto /usr/lib64/python2.6/site-packages/
```
After that, the salt-master starts normally.

**Comment by s0undt3ch** [comment]
@terminalmage can we get our COPR repo's to host a more recent PyCrypto package?

**Comment by dmurphy18** [comment]
This item shall be fixed in the release for 2015.8, see #25878
